recipes/libspnav/LICENSE

```diff
@@ -0,0 +1,23 @@
+Copyright (C) 2007-2018 John Tsiombikas <nuclear@member.fsf.org>
+
+Redistribution and use in source and binary forms, with or without
+modification, are permitted provided that the following conditions are met:
+
+1. Redistributions of source code must retain the above copyright notice, this
+   list of conditions and the following disclaimer.
```

Comment:
I'll open an upstream issue to ask them to include `LICENSE` in the release tarballs for future releases.

Reply:
Nevermind! This is already done upstream in https://github.com/FreeSpacenav/libspnav/commit/8ff8e254ab0782dcc91455c621bf5a231b7767b1 .

Reply:
so we should keep the license in the recipe?

Reply:
Yes, you did everything right here! Only for future releases we'll be able to remove the file from the feedstock if/when it's already in the tarball.